From 6bc30b5cac3e3a7f3cc7a6821f7b9cde5b74ab54 Mon Sep 17 00:00:00 2001
From: Eilo <eilo2518@gmail.com>
Date: Tue, 8 May 2012 00:36:53 -0500
Subject: [PATCH 3/3] Milestone5.0

---
 README |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/README b/README
index 9fdca1d..216a24d 100644
--- a/README
+++ b/README
@@ -51,6 +51,8 @@ Milestone3.0: Arreglos a Spells, sobre todo para estabilidad del pve y del
               pvp dentro de los reinos.
 Milestone4.0: Arreglos en quests, y cadenas de mision, esto incluye scripts
               en c++ y SmartAI.
+Milestone5.0: Traducciones, e inclusion de varios dentro de tablas y strings
+              del emulador, ya sea en c++ o en sqls.
 
 
 ##################################
@@ -315,6 +317,8 @@ Milestone4.0: Arreglos en quests, y cadenas de mision, esto incluye scripts
 #################
 # Traducciones: #
 #################
+5.1  - 29/12/2011: Traducciones de eswow2 de varias tablas de world, actualizadas por Eilo
+5.2  - 08/12/2011: Adecuaciones de nombres de 2 game_tele por Eilo
 
 
 #############################
-- 
1.7.8.msysgit.0

