From 5be838d37ad0ef6c5700d2953faeea7fc0784539 Mon Sep 17 00:00:00 2001
From: Eilo <eilo2518@gmail.com>
Date: Tue, 1 May 2012 13:08:35 -0500
Subject: [PATCH 13/13] 2.1.4 TrialOfTheCrusader: AnubArak aggrea a 20y

---
 .../TrialOfTheCrusader/boss_anubarak_trial.cpp     |   10 ++++++++++
 1 files changed, 10 insertions(+), 0 deletions(-)

diff --git a/src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheCrusader/boss_anubarak_trial.cpp b/src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheCrusader/boss_anubarak_trial.cpp
index e81dc97..a814193 100755
--- a/src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheCrusader/boss_anubarak_trial.cpp
+++ b/src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheCrusader/boss_anubarak_trial.cpp
@@ -168,6 +168,11 @@ public:
         uint64 m_uiTargetGUID;
         uint8  m_uiScarabSummoned;
 
+        void InitializeAI()
+        {
+            me->SetReactState(REACT_AGGRESSIVE);
+        }
+
         void Reset()
         {
             m_uiFreezeSlashTimer = 15*IN_MILLISECONDS;
@@ -209,6 +214,11 @@ public:
                 DoScriptText(SAY_INTRO, me);
                 m_bIntro = false;
             }
+            if (me->IsWithinDistInMap(who, 20.0f))
+            {
+                me->SetReactState(REACT_AGGRESSIVE);
+                me->SetInCombatWithZone();
+            }
         }
 
         void JustReachedHome()
-- 
1.7.8.msysgit.0

