From 1caf1532ceee21ea6687ca8d13c4081bd914f2e8 Mon Sep 17 00:00:00 2001
From: Eilo <eilo2518@gmail.com>
Date: Tue, 1 May 2012 15:57:44 -0500
Subject: [PATCH 13/37] 2.1.4 TrialOfTheCrusader: AnubArak aggrea a 20y

---
 .../TrialOfTheCrusader/boss_anubarak_trial.cpp     |   12 +++++++++++-
 1 files changed, 11 insertions(+), 1 deletions(-)

diff --git a/src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheCrusader/boss_anubarak_trial.cpp b/src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheCrusader/boss_anubarak_trial.cpp
index e81dc97..61c3718 100755
--- a/src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheCrusader/boss_anubarak_trial.cpp
+++ b/src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheCrusader/boss_anubarak_trial.cpp
@@ -168,6 +168,11 @@ public:
         uint64 m_uiTargetGUID;
         uint8  m_uiScarabSummoned;
 
+        void InitializeAI()
+        {
+            me->SetReactState(REACT_AGGRESSIVE);
+        }
+
         void Reset()
         {
             m_uiFreezeSlashTimer = 15*IN_MILLISECONDS;
@@ -202,13 +207,18 @@ public:
             }
         }
 
-        void MoveInLineOfSight(Unit* /*who*/)
+        void MoveInLineOfSight(Unit* who)
         {
             if (!m_bIntro)
             {
                 DoScriptText(SAY_INTRO, me);
                 m_bIntro = false;
             }
+            if (me->IsWithinDistInMap(who, 20.0f))
+            {
+                me->SetReactState(REACT_AGGRESSIVE);
+                me->SetInCombatWithZone();
+            }
         }
 
         void JustReachedHome()
-- 
1.7.8.msysgit.0

