From b77ff6e89d64adc06811c090be773cbf50f244d4 Mon Sep 17 00:00:00 2001
From: Eilo <eilo2518@gmail.com>
Date: Wed, 14 Mar 2012 15:32:32 -0500
Subject: [PATCH 23/93] 2.2.15 ICC: Marrowgar aggrea a 25y

---
 .../IcecrownCitadel/boss_lord_marrowgar.cpp        |    9 +++++++++
 1 files changed, 9 insertions(+), 0 deletions(-)

diff --git a/src/server/scripts/Northrend/IcecrownCitadel/boss_lord_marrowgar.cpp b/src/server/scripts/Northrend/IcecrownCitadel/boss_lord_marrowgar.cpp
index 71b014e..a7e9224 100755
--- a/src/server/scripts/Northrend/IcecrownCitadel/boss_lord_marrowgar.cpp
+++ b/src/server/scripts/Northrend/IcecrownCitadel/boss_lord_marrowgar.cpp
@@ -97,6 +97,11 @@ class boss_lord_marrowgar : public CreatureScript
                 me->ApplySpellImmune(0, IMMUNITY_MECHANIC, MECHANIC_GRIP, true);
             }
 
+            void InitializeAI()
+            {
+                me->SetReactState(REACT_AGGRESSIVE);
+            }
+
             void Reset()
             {
                 _Reset();
@@ -147,6 +152,10 @@ class boss_lord_marrowgar : public CreatureScript
                     Talk(SAY_ENTER_ZONE);
                     _introDone = true;
                 }
+                else if (me->IsWithinDistInMap(who, 20.0f))
+                {
+                    me->SetInCombatWithZone();
+                }
             }
 
             void UpdateAI(uint32 const diff)
-- 
1.7.8.msysgit.0

