From 8bde8e42f74b51b003bb59594fdf691bd679e22b Mon Sep 17 00:00:00 2001
From: Eilo <eilo2518@gmail.com>
Date: Wed, 14 Mar 2012 17:04:14 -0500
Subject: [PATCH 27/93] 2.2.19 ICC: Festergut aggrea al acercarse

---
 .../Northrend/IcecrownCitadel/boss_festergut.cpp   |   28 ++++++++++---------
 1 files changed, 15 insertions(+), 13 deletions(-)

diff --git a/src/server/scripts/Northrend/IcecrownCitadel/boss_festergut.cpp b/src/server/scripts/Northrend/IcecrownCitadel/boss_festergut.cpp
index 81c5038..8bd46e1 100755
--- a/src/server/scripts/Northrend/IcecrownCitadel/boss_festergut.cpp
+++ b/src/server/scripts/Northrend/IcecrownCitadel/boss_festergut.cpp
@@ -92,7 +92,6 @@ class boss_festergut : public CreatureScript
             void Reset()
             {
                 _Reset();
-                me->SetReactState(REACT_DEFENSIVE);
                 events.ScheduleEvent(EVENT_BERSERK, 300000);
                 events.ScheduleEvent(EVENT_INHALE_BLIGHT, urand(25000, 30000));
                 events.ScheduleEvent(EVENT_GAS_SPORE, urand(20000, 25000));
@@ -113,20 +112,23 @@ class boss_festergut : public CreatureScript
 
             void EnterCombat(Unit* who)
             {
-                if (!instance->CheckRequiredBosses(DATA_FESTERGUT, who->ToPlayer()))
+                if (CheckInRoom())
                 {
-                    EnterEvadeMode();
-                    instance->DoCastSpellOnPlayers(LIGHT_S_HAMMER_TELEPORT);
-                    return;
-                }
+                    if (!instance->CheckRequiredBosses(DATA_FESTERGUT, who->ToPlayer()))
+                    {
+                        EnterEvadeMode();
+                        instance->DoCastSpellOnPlayers(LIGHT_S_HAMMER_TELEPORT);
+                        return;
+                    }
 
-                me->setActive(true);
-                Talk(SAY_AGGRO);
-                if (Creature* gasDummy = me->FindNearestCreature(NPC_GAS_DUMMY, 100.0f, true))
-                    _gasDummyGUID = gasDummy->GetGUID();
-                if (Creature* professor = ObjectAccessor::GetCreature(*me, instance->GetData64(DATA_PROFESSOR_PUTRICIDE)))
-                    professor->AI()->DoAction(ACTION_FESTERGUT_COMBAT);
-                DoZoneInCombat();
+                    me->setActive(true);
+                    Talk(SAY_AGGRO);
+                    if (Creature* gasDummy = me->FindNearestCreature(NPC_GAS_DUMMY, 100.0f, true))
+                        _gasDummyGUID = gasDummy->GetGUID();
+                    if (Creature* professor = ObjectAccessor::GetCreature(*me, instance->GetData64(DATA_PROFESSOR_PUTRICIDE)))
+                        professor->AI()->DoAction(ACTION_FESTERGUT_COMBAT);
+                    DoZoneInCombat();
+                }
             }
 
             void JustDied(Unit* /*killer*/)
-- 
1.7.8.msysgit.0

