From 1da4b83a0d76f789f2b2a7fc3a82ee65210007ff Mon Sep 17 00:00:00 2001
From: Eilo <eilo2518@gmail.com>
Date: Fri, 4 May 2012 06:23:28 -0500
Subject: [PATCH 55/91] 2.2.54 ICC: Las oozes de putri y las ice spheres de lk
 deben moverse

---
 .../TargetedMovementGenerator.cpp                  |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/src/server/game/Movement/MovementGenerators/TargetedMovementGenerator.cpp b/src/server/game/Movement/MovementGenerators/TargetedMovementGenerator.cpp
index fdff5a9..b7bf30d 100755
--- a/src/server/game/Movement/MovementGenerators/TargetedMovementGenerator.cpp
+++ b/src/server/game/Movement/MovementGenerators/TargetedMovementGenerator.cpp
@@ -137,6 +137,9 @@ bool TargetedMovementGeneratorMedium<T,D>::Update(T &owner, const uint32 & time_
     // prevent movement while casting spells with cast time or channel time
     if (owner.HasUnitState(UNIT_STATE_CASTING))
     {
+        if (owner.GetEntry() == 37562 || owner.GetEntry() == 37697 || owner.GetEntry() == 36633)
+            return true;
+
         if (!owner.IsStopped())
             owner.StopMoving();
         return true;
-- 
1.7.8.msysgit.0

