From 7a9a522cac103e7bb36d8485bba2120d7f3ad843 Mon Sep 17 00:00:00 2001
From: Eilo <eilo2518@gmail.com>
Date: Sat, 21 Apr 2012 13:23:19 -0500
Subject: [PATCH 19/19] Milestone0.0

---
 README |   23 ++++++++++++++++++++++-
 1 files changed, 22 insertions(+), 1 deletions(-)

diff --git a/README b/README
index 87e8691..06dc29e 100644
--- a/README
+++ b/README
@@ -37,6 +37,11 @@ parches y/o c
 #########
 #ReanEmu#
 #########
+Milestone0.0: Reiniciando el repo y readaptando cosas varias, inclusion de nuevos
+              developers en git y la inclusion de texto de planeacion para union 
+              eficiente de codigo reestructurando modo de trabajo y organizando 
+              carpetas, branches, etc, adicion de codigo utilitario en general 
+              para complemento del nucleo y funciones de soporte.
 
 
 ##################################
@@ -47,6 +52,23 @@ parches y/o c
 ###############
 # Adicionales #
 ###############
+0.1  - 28/10/2011: Archivo sqlcombinado.BAT para unificar sqls de actualizacion por Eilo
+0.2  - 24/11/2011: Añadido sql TDB11.46
+0.4  - 28/10/2011: Pvp Ranks, honor y titulos original por Spgm, adecuaciones varias por Eilo
+0.5  - 10/04/2012: AHBot por Aokrommes
+0.9  - 14/10/2011: Cuentas VIP orginal por Locknes, modificado por Eilo
+0.10 - 11/02/2012: Rates aumentados Fin de Semana (weekend) por Kline, modificado por Eilo
+0.12 - 29/11/2011: TrinityJail original de Warhead, añadido por Hispano, ajustes por Eilo
+0.13 - 20/01/2012: Autenticacion con cata 4.0.6a 4.2.2 y 4.3.3 por Eilo
+0.18 - 20/04/2012: Wowarmory con chart de arenas y 3d original por Shadez, adecuaciones por Eilo
+0.19 - 19/04/2012: SOAP comandos de informacion del server por Eilo
+0.20 - 24/04/2012: CharacterHistory por Codels
+0.21 - 24/04/2012: MailHistory por Codels
+0.22 - 26/04/2012: BossHistory log db y RaidInfo anuncios por Eilo
+0.23 - 08/04/2012: Arenas anuncios de cola por Oceanor
+0.24 - 10/04/2012: Sanctuary zonas custom por Oceanor
+0.25 - 10/04/2012: FFA Pvp zonas custom por Oceanor
+0.26 - 29/04/2012: Arena Espectador por Eilo
 
 
 ###############
-- 
1.7.8.msysgit.0

