From 442671d68c98fb43d2768d706aa3e7abcfedeec7 Mon Sep 17 00:00:00 2001
From: Eilo <eilo2518@gmail.com>
Date: Sat, 21 Apr 2012 13:23:19 -0500
Subject: [PATCH 20/20] Milestone0.0

---
 README |   25 ++++++++++++++++++++++++-
 1 files changed, 24 insertions(+), 1 deletions(-)

diff --git a/README b/README
index 87e8691..9e6a047 100644
--- a/README
+++ b/README
@@ -37,6 +37,11 @@ parches y/o c
 #########
 #ReanEmu#
 #########
+Milestone0.0: Reiniciando el repo y readaptando cosas varias, inclusion de nuevos
+              developers en git y la inclusion de texto de planeacion para union 
+              eficiente de codigo reestructurando modo de trabajo y organizando 
+              carpetas, branches, etc, adicion de codigo utilitario en general 
+              para complemento del nucleo y funciones de soporte.
 
 
 ##################################
@@ -47,7 +52,25 @@ parches y/o c
 ###############
 # Adicionales #
 ###############
-
+0.1  - 28/10/2011: Archivo sqlcombinado.BAT para unificar sqls de actualizacion por Eilo
+0.2  - 24/11/2011: Añadido sql TDB11.46
+0.3  - 20/04/2012: Wowarmory con chart de arenas y 3d original por Shadez, adecuaciones por Eilo
+0.4  - 28/10/2011: Pvp Ranks, honor y titulos original por Spgm, adecuaciones varias por Eilo
+0.5  - 19/04/2012: Comandos de informacion del server por Eilo
+0.6  - 10/04/2012: AHBot por Aokrommes
+0.9  - 14/10/2011: Cuentas VIP orginal por Locknes, modificado por Eilo
+0.10 - 11/02/2012: Rates aumentados Fin de Semana (weekend) por Kline, modificado por Eilo
+0.12 - 29/11/2011: TrinityJail original de Warhead, añadido por Hispano, ajustes por Eilo
+0.13 - 20/01/2012: Autenticacion con cata 4.0.6a 4.2.2 y 4.3.3 por Eilo
+0.18 - 01/11/2011: DoCompleteAchievement reañadida al nucleo como funcion activa por Eilo
+0.19 - 04/11/2011: DoAttackerAreaInCombat reañadida al nucleo para soporte por Eilo
+0.20 - 17/11/2011: DoAttackerGroupInCombat reañadida al nucleo para soporte por Eilo
+0.21 - 17/11/2011: _DoAggroPulse funcion y inFightAggroCheck_Timer reañadida para soporte por Eilo
+0.22 - 21/11/2011: UnsummonSpecificCreaturesNearby funcion de soporte añadida por Eilo
+0.23 - 30/12/2011: HasAchieved funcion de soporte reañadida por Eilo
+0.24 - 30/12/2011: FindNearestPlayer y GetNearestPlayersList reañadidas por Eilo
+0.25 - 30/12/2011: setRegeneratingHealth funcion de soporte reañadida por Eilo
+0.26 - 30/12/2011: SetImmuneToPushPullEffects funcion de soporte reañadida por Eilo
 
 ###############
 # Jugabilidad #
-- 
1.7.8.msysgit.0

